run-tests.py

```diff
@@ -11,6 +11,15 @@
 PROJECT_DIR = os.path.dirname(__file__)
 subprocess.call(['find', PROJECT_DIR, '-name', '*.pyc', '-delete'])
 
+# Check for style
+config_file = os.path.join(PROJECT_DIR, 'flake8.cfg')
+# These paths should all pass PEP-8 checks
+paths_to_check = [
+    os.path.join(PROJECT_DIR, 'server', 'pulp', 'server', 'tasks'),
+    os.path.join(PROJECT_DIR, 'server', 'test', 'unit', 'server')]
```

Comment:
Can this be a list of strings instead? That would be easier to edit as we fix stuff, even if it's just like

```python
paths_to_check [os.path.join(PROJECT_DIR, partial_path) for partial_path in list_of_pep8_dirs)]
```